vars/debian-10.yml

```diff
@@ -19,6 +19,8 @@ __peering_manager_other_packages:
   - libjpeg-dev
   - graphviz
   - libssl-dev
+  - acl
```

Comment:
I don't think this package is needed, is it?

Reply:
tl;dr: When the connecting user is unprivileged and the become user is unprivileged as well, Ansible relies on ACLs to set permissions. This requires the `acl` package.

`TASK [peering-manager : create postgresql database] ************************************************************************************************************************************************************
fatal: [pm-ansible.westeurope.cloudapp.azure.com]: FAILED! => {"msg": "Failed to set permissions on the temporary files Ansible needs to create when becoming an unprivileged user (rc: 1, err: chmod: invalid mode: ‘A+user:postgres:rx:allow’\nTry 'chmod --help' for more information.\n}). For information on working around this, see https://docs.ansible.com/ansible/2.11/user_guide/become.html#becoming-an-unprivileged-user"}`
The link in the error message is broken, this one is correct: https://docs.ansible.com/ansible/latest/user_guide/become.html#risks-of-becoming-an-unprivileged-user

Reply:
Understood but this is not up to this role to do this. Users can change this with their own Ansible setup. Ansible running as unprivileged user is not everyone's use case and we should not bother other users with this.